converter/interactive.py

```diff
@@ -1,18 +1,46 @@
-from .converter import celsius_to_fahrenheit, fahrenheit_to_celsius
+from converter.converter import celsius_to_fahrenheit, fahrenheit_to_celsius, ConverterContext
+from converter.utils import is_float
 
-CELSIUM_INPUT = ['celsius', 'cels', 'cel', '-c', 'c']
+CELSIUS_INPUT = ['celsius', 'cels', 'cel', '-c', 'c']
 FAH_INPUT = ['fahrenheit', 'fahren', 'far', '-f', 'f']
 QUIT_INPUT = ['quit', 'q', 'exit']
 
 
-def get_interactive_function(answer: str) -> tuple:
+def get_interactive_function(answer: str) -> callable:
     if answer in QUIT_INPUT:
         quit()
-    if answer in CELSIUM_INPUT:
-        return celsius_to_fahrenheit, \
-               input("Provide temperature in Celsius: ")
+    if answer in CELSIUS_INPUT:
+        return celsius_to_fahrenheit
 
     if answer in FAH_INPUT:
-        return fahrenheit_to_celsius, \
-               input("Provide temperature in Fahrenheit: ")
+        return fahrenheit_to_celsius
     raise ValueError("Provide correct scale")
+
+
+def get_interactive_temperature() -> float:
```

Comment:
@MariaMaximova, I think it is not used anymore. Can you please check and remove the function if it is true?

Reply:
Done

Reply:
[Deleted](https://github.com/MariaMaximova/convert_temp/pull/19/commits/783a509aa1c26b0826ad164b19370b38f1a70377#diff-f568b8865793141584f41fd001fd963617a3fab0ca0210dd65d5dffe8f7dc3e2)